.github/workflows/ci_task_branch.yml

```diff
@@ -1,19 +1,31 @@
-name: 'task branch pipeline'
+name: 'Task Branch Pipeline'
 on:
   pull_request:
     types: [ opened, reopened, synchronize ]
 
-jobs: 
+jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Build container
-        uses: docker/build-push-action@v1
+      - name: Login to Quay.io
+        uses: docker/login-action@v2
         with:
+          registry: quay.io
           username: ${{ secrets.QUAY_ROBOT_USER_NAME }}
           password: ${{ secrets.QUAY_ROBOT_TOKEN }}
-          registry: quay.io
-          repository: ukhomeofficedigital/hocs-search
-          tag_with_sha: true
-
+      - name: Calculate Container Tags
+        id: meta
+        uses: docker/metadata-action@v4
```

Comment:
Docker metadata plugin will provide the metdata that we put against our images so the tag we use to reference that image from our deployments and the labels inside the image manifest
https://quay.io/repository/ukhomeofficedigital/hocs-search/manifest/sha256:d4d1e9d653414db00ea599fb523480eb625b95a4124450c720fa423d72ecbb09